bin/zowe-setup-certificates.env

```diff
@@ -23,6 +23,12 @@ EXTERNAL_CERTIFICATE_AUTHORITIES=
 # optional - Public certificates of z/OSMF - multiple certificates 
 # delimited with space has to be enclosed with quotes ("cer1 cer2")
 ZOSMF_CERTIFICATE=
+# Enable SSO during setup
+SETUP_APIML_SSO=true
```

Comment:
Is this not already handled elsewhere? Lets coordinate to make sure our story on defaults is shared across zowe.